### PR TITLE
Investigate/118

### PR DESCRIPTION
**Pull request by YouvaB**
Prettier kind of messed up the git diff, but the actual difference is on line 490-492. Just need to add

```
if (errorResult.valid) {
   return createReturnObject();
}
```

### CODE REVIEW

File: tests/testPatternValidation.js [hunk 1]
```diff
@@ -1,362 +1,377 @@
...
+    test.ok(!errors.valid);
+    // Check if invalid model for all of oneOf references
+    var result = false;
+    for (const error of errors.errors) {
+      result =
+        error.message ===
+        "Error: contactMechanism is not a valid target for a oneOf";
+    }
+    test.ok(!result);
+    test.done();
```
Comment (YouvaB) [inline tests/testPatternValidation.js]: This is wrong. Will fix in a bit.

**Comment by brodyd** [comment]
Thanks fore your efforts but the wholesale refactoring of style means that it is very difficult to tell what you have actually done.  I'd be much more likely to take this in if you don't change the coding style.

**Comment by YouvaB** [comment]
> 
> 
> Thanks for your efforts but the wholesale refactoring of style means that it is very difficult to tell what you have actually done. I'd be much more likely to take this in if you don't change the coding style.

Yeah I figured, prettier kind of messed the whole thing up. I'll rewrite my changes without prettier tomorrow, but it isn't much changes.

**Comment by YouvaB** [comment]
@brodyd Hello! Just removed all the prettier reformatting, now the commit diff should only show what I wrote. Hopefully, this is better!

**Pull request merged**